examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,629 @@
+    "    - type: MapToFields\n",
+    "      config:\n",
+    "        language: python\n",
+    "        append: true\n",
+    "        fields:\n",
+    "          is_adult: \"age >= 18\"\n",
+    "        drop:\n",
+    "          - age\n",
+    "    - type: LogForTesting\n",
+    "'''\n",
+    "save_to_file(pipeline, 'pipelines/pipeline-map-03.yaml')\n",
+    "! python -m apache_beam.yaml.main --pipeline_spec_file=pipelines/pipeline-map-03.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also create simple UDFs (User Defined Functions) using Python or other languages. In the example below we add a field `random_number` which value is a random number not bigger than the age of the person."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "pipeline = '''\n",
+    "pipeline:\n",
+    "  type: chain\n",
+    "  transforms:\n",
+    "    - type: ReadFromCsv\n",
+    "      config:\n",
+    "        path: data/people.csv\n",
+    "    - type: MapToFields\n",
+    "      config:\n",
+    "        language: python\n",
+    "        append: true\n",
+    "        fields:\n",
+    "          random_number:\n",
+    "            callable: |\n",
+    "              import random\n",
+    "              def my_mapping(row):\n",
+    "                return random.randrange(row.age)\n",
+    "    - type: LogForTesting\n",
+    "'''\n",
+    "save_to_file(pipeline, 'pipelines/pipeline-map-04.yaml')\n",
+    "! python -m apache_beam.yaml.main --pipeline_spec_file=pipelines/pipeline-map-04.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Beam YAML has EXPERIMENTAL ability to do aggregations to group and combine values across records. The is accomplished via the `Combine` transform type. Currently `Combine` needs to be in the `yaml_experimental_features` option (see the bottom of the pipeline) to use this transform.\n",
+    "\n",
+    "In this example we'll aggregate our records based on the `is_adult` classification. We'll calculate an average age for each of the groups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "pipeline = '''\n",
+    "pipeline:\n",
+    "  type: chain\n",
+    "  transforms:\n",
+    "    - type: ReadFromCsv\n",
+    "      config:\n",
+    "        path: data/people.csv\n",
+    "    - type: MapToFields\n",
+    "      config:\n",
+    "        language: python\n",
+    "        append: true\n",
+    "        fields:K\n",
+    "          is_adult: \"age >= 18\"\n",
+    "    - type: Combine\n",
+    "      config:\n",
+    "        group_by: is_adult\n",
+    "        combine:\n",
+    "          total:\n",
+    "            value: age\n",
+    "            fn:\n",
+    "              type: mean\n",
```

Comment:
nit: This can be simplified.

```suggestion
    "    - type: Combine\n",
    "      config:\n",
    "        group_by: is_adult\n",
    "        combine:\n",
    "          total:\n",
    "            value: age\n",
    "            fn: mean\n",
```

Reply:
Thanks, adjusted.